cordova-lib/src/cordova/util.js

```diff
@@ -297,7 +297,7 @@ function ensurePlatformOptionsCompatible (platformOptions) {
     }).map(function (arg) {
         return opts[arg] === true ?
             '--' + arg :
-            '--' + arg + '="' + opts[arg].toString() + '"';
+            '--' + arg + '=' + opts[arg].toString();
```

Comment:
Because these quotes distorts arguments values e.g. `['--appx=uap']` becomes `['--appx="uap"']` and then (after parsing by `nopt` in platform) - `{appx: "\"uap"\"}` - notice that quotes now included as a part of `appx` option value.